configcatclient/rolloutevaluator.py

```diff
@@ -179,12 +182,14 @@ def evaluate(self, key, user, default_value, config):
                 bucket += rollout_percentage_item.get(self.PERCENTAGE, 0)
                 if hash_val < bucket:
                     percentage_value = rollout_percentage_item.get(self.VALUE)
+                    variation_id = rollout_percentage_item.get(self.VARIATION_ID, default_variation_id)
                     log.info('Evaluating %% options. Returning %s' % percentage_value)
-                    return percentage_value
+                    return percentage_value, variation_id
 
         def_value = setting_descriptor.get(self.VALUE, default_value)
```

Comment:
For me it is unclear what's the difference btwn `def_value` and `default_value`. I suggest a proper naming.